### PR TITLE
Packaging with Wix to create .msi installer 

### PR DESCRIPTION
**Pull request by Arslan-Siraj**
- wix base .msi installer (user will get one .msi file and install on PC)
- App shortcut: Desktop and inside app folder
- OpenMS license added
- Readme created inside github action
- How for new Apps in github action?
               -  user has to write the App_name
               -  copy icon of their app to asset folder 
               -  placed and copied everything needed in streamlit_exe folder (folder exist in github action) which will generate the wixobj automatically

### CODE REVIEW

**Review by t0mdavid-m — COMMENTED** [review]
Looks good to me in general! Excellent implementation and much better from a user perspective.

One thing I noticed is that when the streamlit server is initially launched, the web browser is not launched automatically. This is not ideal from a user perspective. If the App is subsequently opened however, everything works as usual (i.e. the browser gets launched automatically.

As a workaround I would suggest (not sure if that is easily possible) to not launch the app directly but instead display a message that the installation was successful.

I also have another feature request: Could we launch the Streamlit server minimized? This way users wont even have to look at the console window.

File: .github/workflows/build-windows-executable-app.yaml [hunk 1]
```diff
@@ -254,24 +263,130 @@ jobs:
...
 
-    - name: Delete OpenMS bin artifact
-      uses: geekyeggo/delete-artifact@v5
-      with:
-          name: openms-package
-          
-    - name: Compress streamlit_exe folder to OpenMS-App.zip
+    - name: Generate Readme.txt
+      run: |
+        echo "Welcome to ${{ env.APP_NAME }}!" > streamlit_exe/Readme.txt
```
Comment (jpfeuffer) [inline .github/workflows/build-windows-executable-app.yaml:275]: Could you use HereDocs instead of the many echos?

**Comment by Arslan-Siraj** [comment]
> Looks good to me in general! Excellent implementation and much better from a user perspective.
> 
> One thing I noticed is that when the streamlit server is initially launched, the web browser is not launched automatically. This is not ideal from a user perspective. If the App is subsequently opened however, everything works as usual (i.e. the browser gets launched automatically.
> 
> As a workaround I would suggest (not sure if that is easily possible) to not launch the app directly but instead display a message that the installation was successful.
> 
> I also have another feature request: Could we launch the Streamlit server minimized? This way users wont even have to look at the console window.

- If installed in the program files folder, it will not open in the default browser directly. I think some security checks are still missing or might this behaviour exist. If use other paths to install the app, it works, no need for a browser selection.

- That's what I want with launching the app—not the console open, basically in the shortcut we are calling .bat file, which is available in the installed folder --> which took to run the app. (not sure the best way to bind this)

**Comment by t0mdavid-m** [comment]
Works like a charm. Thank you for your great contribution!

**Pull request merged**